docs/user/Multi-Tenancy.md

```diff
@@ -122,20 +132,20 @@ Step 4. Add any additional CAS customizations for tenants as needed and then run
     playbooks/playbook.yaml --tags "multi-tenancy,offboard"
   ```
 
-### Example Command to Uninstall SAS Viya with Multi-Tenancy
+## Uninstall SAS Viya with Multi-Tenancy Enabled Deployment
+Before you run uninstall command make sure to run offboard command for any onboarded tenants.
+
+### Run the following command to uninstall deployment
 
   ```bash
   ansible-playbook \
     -e BASE_DIR=$HOME/deployments \
     -e KUBECONFIG=$HOME/deployments/.kube/config \
     -e CONFIG=$HOME/deployments/dev-cluster/dev-namespace/ansible-vars.yaml \
     -e JUMP_SVR_PRIVATE_KEY=$HOME/.ssh/id_rsa \
-    playbooks/playbook.yaml --tags "baseline,viya,multi-tenancy,uninstall"
+    playbooks/playbook.yaml --tags "baseline,viya,uninstall"
```

Comment:
Just clarifying here. The `viya,uninstall` also does the `multi-tenancy,uninstall` but if needed the customer can also perform just the `multi-tenancy,uninstall` if needed without removing viya. Is that correct?

Reply:
No, earlier `multi-tenancy,uninstall` removed the cas directories created for tenants but JK found out that for a deployment with onboarded tenants offboarding is necessary to remove database resources. So for better cleanup of resources on a MT enabled deployment, you will have to run offboard before uninstall. Multi-tenancy is enabled in Viya deployment so to remove multi-tenancy user will have to uninstall viya.

Reply:
Hi Ritika, are the latest changes to the multi-tenancy uninstall behavior you described in the paragraph above expressed in your .md file, in particular the last sentence?

Reply:
lol. Yeah she states that on line 135-136 of this doc.

Reply:
Yes the uninstall section in README says to run offboard command first before uninstall.

Reply:
ok, I was asking about this in particular since it seemed helpful to know. "Multi-tenancy is enabled in Viya deployment so to remove multi-tenancy user will have to uninstall viya."

Reply:
Ah no it doesn't say about the last line, but with uninstall you pretty much are deleting the whole deployment not just multi-tenancy so didn't specify it explicitly.